.github/workflows/build-and-publish-docker-image.yml

```diff
@@ -12,23 +11,33 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v3
-
+      - name: Set up QEMU
```

Comment:
What is QEMU required for? If it's a cross-platform thing could we achieve the same result more efficiently by locking this workflow to a specific runner image? https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
It's a good question. I added this and the buildx action without much consideration because somewhere recommended explicitly doing this when building the docker images to fix the versions used under the hood. AFAICT, this is about building images for different platforms: For example, it is used to create ARM64 images while running in an x86 Linux environment. I don't think this is the same as using cross platform runners, although maybe the same thing can be achieved that way. Currently, however, we are not specifying the target platform, so maybe we should be?

Reply:
Ah I see why you need QEMU now. buildx is building images for all target platforms.